content/06.methods.md

```diff
@@ -502,4 +510,7 @@ Next, we used multivariate cox (proportional hazards) regression analysis [@url:
 |CDS from GENCODE v27 annotation                |GENCODE                        |https://www.gencodegenes.org/human/release_27.html                     |
 |PFAM domains and locations                    |UCSC                        |http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/pfamDesc.txt.gz; https://pfam.xfam.org/family/PF07714                      |
 |BSgenome.Hsapiens.UCSC.hg38 annotations                   |Bioconductor                        |https://bioconductor.org/packages/release/data/annotation/html/BSgenome.Hsapiens.UCSC.hg38.html                     |
+|gnomAD v2.1.1                    |Genome Aggregation Database                        |https://gnomad.broadinstitute.org/downloads#v2-liftover-variants                     |
+|KEGG MMR gene set                    |BROAD Institute                        |https://www.gsea-msigdb.org/gsea/msigdb/download_geneset.jsp?geneSetName=KEGG_MISMATCH_REPAIR&fileType=TSV                     |
```

Comment:
Do we want to add a date to this one as well, since it does not appear to be versioned?

Reply:
woahhh - if I use this link instead: https://www.gsea-msigdb.org/gsea/msigdb/download_geneset.jsp?geneSetName=KEGG_MISMATCH_REPAIR then it downloads a versioned text file. Win win!